README.md

```diff
@@ -48,56 +53,6 @@ make quake
 
 The usage and limitations of Doom and Quake demo are listed in [docs/demo.md](docs/demo.md).
 
-## riscv-arch-test
```

Comment:
The section should be kept along with revisions for new RISCOF framework.

Reply:
I see.